config/loss/multiclass/ohemcrossentropy.yaml

```diff
@@ -0,0 +1,4 @@
+# @package _global_
```

Comment:
Should we name this file ohemce.yaml or keep full name?

Reply:
I agree less long